tests/ci/run_build.py

```diff
@@ -51,7 +51,7 @@ def run_command(command):
         command,
         bufsize=1,
         stdout=subprocess.PIPE,
-        stderr=subprocess.STDOUT,
+        stderr=subprocess.PIPE,
```

Comment:
IIRC, after the commit (Changed `STDOUT` to `PIPE` ): https://github.com/openhpc/ohpc/commit/783b246304b91c73ae407ec2afc1e967e5406190

Even the package build is successful, the output can't be captured for some reason in openEuler:

https://github.com/openhpc/ohpc/blob/8a1b42c236cb0071839568dcde40e5d39c7844c7/tests/ci/run_build.py#L100-L104

And finally cause the CI failure, so I revert this to make openEuler CI happy (I doubted that this might related to python version).

But I think we can first drop this fix first, and the problem we meet in future we can invistigation deeply.